app/backend/src/couchers/models.py

```diff
@@ -266,6 +262,56 @@ def __repr__(self):
     )
 
 
+class LanguageFluency(enum.Enum):
+    # note that the numbering is important here, these are ordinal
+    say_hello = 1
+    beginner = 2
+    intermediate = 3
+    advanced = 4
+    fluent = 5
+    native = 6
```

Comment:
` = enum.auto()` on all these.

Reply:
The idea is that these are ordinal, so they can be used in filters (e.g. find users whose language ability in Finnish >= `intermediate`).